Automattic/wp-calypso
# Remove "beta testing" item from the menu for all users

#### Changes proposed in this Pull Request

Following the discussion on #56312, we want to remove the "beta testing" item from the menu.

This is the Calypso part of it. The WPCOM part is merged and deployed, which means that we do not see `Beta testing` on `/wp-admin` anymore.

Since Jetpack is only going to be deployed on 2021-11-02, it makes sense to remove only the menu item and keep the page.

#### Testing instructions

Open a site on WP.COM and check that the "beta testing" menu is gone. Verify you can still access it via the `/beta-testing/` page on any site.

Part of #56312.
